photoprism/.env-dist

```diff
@@ -0,0 +1,108 @@
+PHOTOPRISM_TRAEFIK_HOST=photoprism.example.com
+PHOTOPRISM_VERSION=230719
+PHOTOPRISM_INSTANCE=
+TIMEZONE=Etc/UTC
+
+## The following can be named docker volumes OR host mounted absolute paths.
+## Make sure to create these directories before install.
+##
+## Where to store your original media files:
+PHOTOPRISM_ORIGINALS_VOLUME=/tmp/images
+##
+## Optionally, you can add additional media volumes, but be sure to also add
+## them to the `photoprism` service in `docker-compose.yaml` as well.
+## Example:
+# PHOTOPRISM_HOST_MEDIA_VOLUME_2=/tmp/more-images
+# PHOTOPRISM_CONTAINER_MEDIA_DIR_2=more-images
+##
+## Optionally, you can add a base folder from which files can be imported
+## to originals, but be sure to uncomment the corresponding line in the 
+## `photoprism` service in `docker-compose.yaml` as well.
+# PHOTOPRISM_HOST_IMPORT_VOLUME=/tmp/import
+
+## Traefik htpasswd encoded authentication
+## (automatically set via `make config`):
+PHOTOPRISM_HTTP_AUTH=
+## Traefik IP whitelist filter to block access to the client interface:
+## (does not affect peer connections)
+PHOTOPRISM_IP_SOURCERANGE=0.0.0.0/0
+
+
+PHOTOPRISM_ADMIN_USER=admin
+# initial admin password (8-72 characters)
+PHOTOPRISM_ADMIN_PASSWORD=password
+# server URL in the format "http(s)://domain.name(:port)/(path)"
+PHOTOPRISM_SITE_URL=http://localhost:2342/
+PHOTOPRISM_SITE_CAPTION=AI-Powered Photos App
+PHOTOPRISM_SITE_DESCRIPTION=
+PHOTOPRISM_SITE_AUTHOR=
+## Run/install on first startup (options: update https gpu tensorflow davfs clitools clean):
+# PHOTOPRISM_INIT=https gpu tensorflow
+## Hardware Video Transcoding:
+# FFmpeg encoder ("software", "intel", "nvidia", "apple", "raspberry")
+# PHOTOPRISM_FFMPEG_ENCODER=software
+# FFmpeg encoding bitrate limit in Mbit/s (default: 50)
+# PHOTOPRISM_FFMPEG_BITRATE=32
+# FFmpeg encoding resolution limit in pixel height (default: 4096)
+# PHOTOPRISM_FFMPEG_RESOLUTION=1920
+## Run as a non-root user after initialization (supported: 0, 33, 50-99, 500-600, and 900-1200):
+PHOTOPRISM_UID=1000
+PHOTOPRISM_GID=1000
+PHOTOPRISM_UMASK=0000
+# MariaDB or MySQL database user password - this should match MARIADB_PASSWORD
+PHOTOPRISM_DATABASE_PASSWORD=insecure
+# this should match PHOTOPRISM_DATABASE_PASSWORD
+MARIADB_PASSWORD=insecure
+MARIADB_ROOT_PASSWORD=insecure
+
+## There's probably very little reason to change any values below here.
+# authentication mode (public, password)
+PHOTOPRISM_AUTH_MODE=password
+# disables HTTPS/TLS even if the site URL starts with https://
+PHOTOPRISM_DISABLE_TLS=true
```

Comment:
this should not be exposed, because it should always be set since its behind traefik.

Reply:
fixed